picard/oauth.py

```diff
@@ -44,41 +44,99 @@ class OAuthManager(object):
 
     def __init__(self, webservice):
         self.webservice = webservice
+        self.persist= config.persist
```

Comment:
Missing space

Reply:
Fixed in dc5d383f3cbef49fb0605a67b4832b6f5d8d6c52